### DISCUSSION TITLE
Works on Ubuntu?

### DISCUSSION THREAD

**Discussion by Sshahar** [Senshi111/debian-hyprland-hyprdots#13] [Q&A] [answered]
Tried installing it on Ubuntu 22.04, but no visible changes. Do I need to config it somewhere?

**Comment by Senshi111** [comment, marked as answer]
> Tried installing it on Ubuntu 22.04, but no visible changes. Do I need to config it somewhere?

Last time when I was trying to install on Ubuntu, it works but i had to add some extra dependencies for hyprland on Ubuntu.
Question is, is hyprland going to build on Ubuntu 22.04 because most of the time it requires lattest dependencies. Try first to build hyprland script and see if its fails on some point.
